app/V1Module/security/AuthorizatorBuilder.php

```diff
@@ -40,14 +41,15 @@ public function build($aclInterfaces, $permissions, $uniqueId): ClassType
         $check->addParameter("privilege")->setType("string");
 
         foreach ($permissions as $i => $rule) {
-            $all = new PhpLiteral(sprintf("%s::ALL", Permission::class));
```

Comment:
Does this change really do the same thing as the original code?

Reply:
It should, since `::ALL` is not used anywhere and the condition was constructed so it always returned false.